Helm: Simplified TLS configuration (#11776)
grafana/mimir: operations/helm/charts/mimir-distributed/CHANGELOG.md

* [CHANGE] Memcached: Set a timeout of `500ms` for the `ruler-storage` cache instead of the default `200ms`. #11231
* [CHANGE] All: Environment variable `JAEGER_REPORTER_MAX_QUEUE_SIZE` is no longer set. Components will use OTel's default value of `2048` unless explicitly configured. You can still configure `JAEGER_REPORTER_MAX_QUEUE_SIZE` if you configure tracing using Jaeger env vars, and you can always set `OTEL_BSP_MAX_QUEUE_SIZE` OTel configuration. #11700
* [CHANGE] Update rollout-operator to latest release, this includes support for `OTEL_` tracing environment variables. #11748 #11794

> * [ENHANCEMENT] TLS config is simplified in a central object which is templated to config as necessary. #11776

Reviewer: ```suggestion
* [ENHANCEMENT] TLS config is simplified in a central object (`tls`) which is templated to config if configured. #11776
```

> * [ENHANCEMENT] TLS config is simplified in a central object which is templated to config as necessary. #11776

Reviewer 2: ```suggestion
* [ENHANCEMENT] TLS configuration is simplified in a central object which is templated to configure as necessary. #11776
```
